packages/eslint-plugin/src/util/misc.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @fileoverview Really small utility functions that didn't deserve their own files
+ */
+
+import { TSESTree, AST_NODE_TYPES } from '@typescript-eslint/typescript-estree';
+import RuleModule from 'ts-eslint';
+
+/**
+ * Check if the context file name is *.ts or *.tsx
+ */
+export function isTypescript(fileName: string) {
```

Comment:
OCD nit - Casing on `Typescript`. As you went with `isDefinitionFile` for the similar util function below, maybe `isTypeScriptFile` would make sense?

Reply:
this is an existing function, I just moved it. Happy to rename it.